index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>gbenga-portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="nav-container">
+                <header >
+                    <nav class="nav-bar">
+                        <a href="#">Gbenga</a>
+                        <img src="./images/Union.png" alt="harmburger">
+                    </nav>
+                </header>  
+
+        </div>
```

Comment:
- [ OPTIONAL ] Kindly, Be careful with blank lines and indentation. Do not add blank lines, spaces, or indentations without a reason.  For readability,
- add blank lines to separate large or logical code blocks.
- add two spaces of indentation. (Do not use the tab key).